server/channels/api4/license.go

```diff
@@ -351,10 +351,8 @@ func requestTrueUpReview(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// True-up is only enabled when telemetry is disabled.
-	// When telemetry is enabled, we already have all the data necessary for true-up reviews to be completed.
-	telemetryEnabled := c.App.Config().LogSettings.EnableDiagnostics
-	if telemetryEnabled != nil && !*telemetryEnabled {
+	// Only report the true up review to CWS if the connection is available.
+	if err := c.App.Cloud().CheckCWSConnection(c.AppContext.Session().UserId); err == nil {
```

Comment:
Just to double check because I don't remember the details on which flow calls which, this will not block the case of air-gapped where they download the true-up data correct? I think that iswhat the rest of this handler does outside of this condition?

Reply:
Correct, the flow is maintained for those that can't report to CWS